crates/server/src/admin/handlers/context.rs

```diff
@@ -1,5 +1,6 @@
 use std::str::FromStr;
 use std::sync::Arc;
+use std::vec;
```

Comment:
is this unused?

Reply:
Doesn't mark as unused but also does not complain if I remove it.

there is below:
```
data: ContextUsers {
      context_users: vec![],
  },
```

Reply:
Ah, yeah that's already in the standard library prelude. No need to import explicitly.